### DISCUSSION TITLE
how could I set up the cron job to run after the previous job completed?

### DISCUSSION THREAD

**Discussion by cqyanbo** [kelektiv/node-cron#888] [Q&A] [answered]
I set up a cron job that runs every three seconds. But I also want them to run one by one, not simultaneously. Are there any ideas out there?

**Comment by intcreator** [comment]
what do you mean by "one by one, not simultaneously?" if it runs every three seconds the job will run every three seconds which sounds like one by one to me

**Reply by sheerlox** [reply]
I think they might be referring to this feature request:

- https://github.com/kelektiv/node-cron/issues/556

**Comment by sheerlox** [comment, marked as answer]
This feature is available since version 3.3.0 :rocket:

For more information, check out the `waitForCompletion` option in the [documentation](https://github.com/kelektiv/node-cron/tree/fa08aa3e0edacdc324381cff33160d49b6db6af0?tab=readme-ov-file#constructor).
